src/components/campaigns/Summary/__specs__/CampaignSummary-spec.js

```diff
@@ -0,0 +1,151 @@
+        expect(props.fundsRaisedAmount).to.equal(9876543)
+      })
+
+      describe('NumberOfDonations', () => {
+        it('renders the NumberOfDonations components with expected props', () => {
+          const wrapper = mount(<CampaignSummary campaignUid='au-123' />)
+          const props = wrapper.find(NumberOfDonations).first().props()
+          expect(props.total).to.equal(6000)
+          expect(props.numberFormat).to.equal('0,0')
+          expect(props.loading).to.equal(false)
+        })
+
+        it('renders NumberOfDonations components with different numberFormats', () => {
+          const wrapper = mount(<CampaignSummary campaignUid='au-123' />)
+          const components = wrapper.find(NumberOfDonations)
+          expect(components.at(0).text()).to.equal('6,000 donations')
+          expect(components.at(1).text()).to.equal('6.0k donations')
+        })
+      })
+
+      describe('FundsRaised', () => {
+        it('renders the FundsRaised components with expected props', () => {
+          const wrapper = mount(<CampaignSummary campaignUid='au-123' />)
+          const props = wrapper.find(FundsRaised).first().props()
+          expect(props.fundsRaised.amount).to.equal(9876543)
+          expect(props.fundsRaised.currency).to.equal('R')
+          expect(props.numberFormat).to.equal('0,0')
+          expect(props.loading).to.equal(false)
+        })
+
+        it('renders FundsRaised components with different numberFormats', () => {
+          const wrapper = mount(<CampaignSummary campaignUid='au-123' />)
+          const components = wrapper.find(FundsRaised)
+          expect(components.at(0).text()).to.equal('9,876,543 raised')
+          expect(components.at(1).text()).to.equal('9.9m raised')
+        })
+      })
+
+      it('renders the TimeToGive components with expected props', () => {
+        const wrapper = mount(<CampaignSummary campaignUid='au-123' />)
+        const props = wrapper.find(TimeToGive).first().props()
+        expect(
+          props.donationEndTime.toString()
+        ).to.equal(
+          moment('2019-02-15T00:00:00.000+00:00').toString()
+        )
+      })
+
+      describe('Goal', () => {
+        it('renders the Goal components with expected props', () => {
+          const wrapper = mount(<CampaignSummary goal={666000} campaignUid='au-123' />)
+          const props = wrapper.find(Goal).first().props()
+          expect(props.goal).to.equal(666000)
+          expect(props.currencySymbol).to.equal('R')
+          expect(props.numberFormat).to.equal('0,0')
+          expect(props.loading).to.equal(false)
+        })
+
+        it('renders Goal components with different numberFormats', () => {
+          const wrapper = mount(<CampaignSummary goal={666000}campaignUid='au-123' />)
+          const components = wrapper.find(Goal)
+          expect(components.at(0).text()).to.equal('666,000 ')
+          expect(components.at(1).text()).to.equal('666.0k ')
```

Comment:
With the small format I think it would be better to use this format: `'0a[.]0'`. It's basically the same with the only difference being that it only shows the `.` if it's needed. So 666,000 would display as 666k, whereas 666,200 would display as 6.2k.